src/redux/reducers.js

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux';
+import app from 'redux/nodes/app/reducer';
```

Comment:
it could be either of those but when we `combineReducers` below the key we pass is the key for the state tree, which I think would still be `app` (`state.app` vs `state.appReducer`) so we'd have to change the end of this file to be

```js
export default combineReducers({
  app: appReducer,
})
```

This shorthand makes it a little easier IMO